src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -113,7 +113,7 @@ export async function validateProjectSelection(project: vscode.QuickPickItem) {
     const classFilePath = compiledJavaFiles[0].fsPath
     const baseCommand = 'javap'
     const args = ['-v', classFilePath]
-    const spawnResult = spawnSync(baseCommand, args, { shell: false, encoding: 'utf-8' })
+    const spawnResult = spawnSync(baseCommand, args, { shell: true, encoding: 'utf-8' })
```

Comment:
Had a customer issue (see below) where setting shell: true for the `mvn` command allowed the shell to find their Maven executable, so will set shell to true here too to pre-empt any issues with the `javap` command. It seems that setting shell to true allows for more permissions so our error rates should drop with these changes.